taipy/core/taipy.py

```diff
@@ -505,7 +505,13 @@ def delete(entity_id: Union[TaskId, DataNodeId, SequenceId, ScenarioId, JobId, C
     raise ModelNotFound("NOT_DETERMINED", entity_id)
 
 
-def get_scenarios(cycle: Optional[Cycle] = None, tag: Optional[str] = None) -> List[Scenario]:
+def get_scenarios(
+    cycle: Optional[Cycle] = None,
+    tag: Optional[str] = None,
+    is_sorted: Optional[bool] = False,
+    descending: Optional[bool] = False,
+    sort_key: Optional[Literal["name", "id", "config_id", "creation_date", "tags"]] = "name",
```

Comment:
```suggestion
    is_sorted: bool = False,
    descending: bool = False,
    sort_key: Literal["name", "id", "config_id", "creation_date", "tags"] = "name",
```
`is_sorted`, `descending`, and `sort_key` don't accept `None` as a possible value, so it should not has the `Optional` type.